src/ui/wxWidgets/YubiMixin.h

```diff
@@ -25,7 +25,7 @@ class wxWindow;
 class YubiMixin
 {
  public:
-  enum {POLLING_INTERVAL = 500}; // mSec
+  enum {POLLING_INTERVAL_MIN = 100, POLLING_INTERVAL = 500, POLLING_INTERVAL_MAX = 60000}; // mSec
```

Comment:
Nit: maybe rename POLLING_INTERVAL to POLLING_INTERVAL_DEFAULT?

Reply:
Changed as suggested.